### ISSUE TITLE
Berry doesn't work for Frankfurt located S3 credential bucket

### ISSUE THREAD

**Issue by Klaxon77** [zalando-stups/berry#13]
Got following error for Frankfurt:

```
taupage-init: INFO: Waiting for berry to download OAuth credentials to /meta/credentials/user.json..
berry: ERROR: Could not read from mint S3 bucket "zalando-stups-mint-607300429032-eu-central-1": 400 Bad Request: The authorization mechanism you have provided is not supported. Please use AWS4-HMAC-SHA256.
```

> AWS4-HMAC-SHA256, also known as "V4," is one of two authentication schemes in S3.
> 
> US-Standard, and many -- but not all -- other regions, also support the other, older scheme, V2
> Since Frankfurt was introduced late in 2014, it apparently does not support V2, which is what this error suggests you are using.
> 
> http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingAWSSDK.html explains how to enable V4 in the various SDKs, assuming you are using an SDK that has that capability.

**Comment by hjacobs** [comment]
Berry now uses boto3 which should support AWS4-HMAC-SHA256, but I'm not sure if the latest Taupage is already rolled out.

**Comment by tuxlife** [comment]
Released with 1.0.12
